packages/@react-spectrum/label/src/StaticField.tsx

```diff
@@ -0,0 +1,200 @@
+
+function StaticField<T extends SpectrumStaticFieldTypes>(props: SpectrumStaticFieldProps<T>, ref: RefObject<HTMLDivElement>) {
+  let {
+    value,
+    formatOptions,
+    labelPosition,
+    labelAlign,
+
+    ...otherProps
+  } = props;
+  let {styleProps} = useStyleProps(otherProps);
+  let labelWrapperClass = classNames(
+    labelStyles,
+    'spectrum-Field',
+    {
+      'spectrum-Field--positionTop': labelPosition === 'top',
+      'spectrum-Field--positionSide': labelPosition === 'side'
+    }
+  );
+
+  return (
+    <div
+      {...filterDOMProps(props)}
+      {...styleProps}
+      className={classNames(labelStyles, 'spectrum-StaticField', {[labelWrapperClass]: props.label}, styleProps.className)}
+      ref={ref}>
+      {props.label &&
+        <Label
+          labelPosition={labelPosition}
+          labelAlign={labelAlign}
+          elementType="span">
+          {props.label}
+        </Label>}
+      <div
+        className={classNames(labelStyles, 'spectrum-Field-wrapper')}>
+        <div
+          className={classNames(labelStyles, 'spectrum-Field-field')}>
+          {Array.isArray(value) && 
+            // @ts-ignore
+            <FormattedStringList value={value} formatOptions={formatOptions as Intl.ListFormatOptions} />}
+
+          {typeof value === 'object' && 'start' in value && typeof value.start === 'number' && typeof value.end === 'number' && 
+            <FormattedNumber value={value as NumberValue} formatOptions={formatOptions as Intl.NumberFormatOptions}  />}
+
+          {typeof value === 'object' && 'start' in value && typeof value.start !== 'number' && typeof value.end !== 'number' && 
+            <FormattedDateRange value={value as DateTimeValue} formatOptions={formatOptions as Intl.DateTimeFormatOptions} />}
+
+          {typeof value === 'number' && 
+            <FormattedNumber value={value} formatOptions={formatOptions as Intl.NumberFormatOptions} />}
+
+          {(typeof value === 'object' && ('calendar' in value || 'hour' in value) || (value instanceof Date))  &&
+            <FormattedDate value={value} formatOptions={formatOptions as Intl.DateTimeFormatOptions} />}
+
+          {typeof value === 'string' && 
+            props.value}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function FormattedStringList<T extends string[]>(props: StringListProps<T>) {
+  let stringFormatter = useListFormatter(props.formatOptions); 
+
+  return (
+    <>{stringFormatter.format(props.value)}</>
+  );
+}
+
+function FormattedNumber<T extends NumberValue>(props: NumberProps<T>) {
+  let numberFormatter = useNumberFormatter(props.formatOptions); 
+  let value = props.value;
+
+  if (typeof value === 'object' && typeof (value as RangeValue<NumberValue>).start === 'number' && typeof (value as RangeValue<NumberValue>).end === 'number') {
+    return <>{numberFormatter.formatRange(value.start, value.end)}</>;
+  }
+
+  return <>{numberFormatter.format(value as number)}</>;
```

Comment:
I don't think the type assertion should be necessary here

```suggestion
  return <>{numberFormatter.format(value)}</>;
```